.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,41 +6,39 @@ labels: 'investigating'
 assignees: ''
 
 ---
-
-<!--
-NOTICE: GitHub is not a mechanism for receiving support under any agreement or SLA. If you require immediate assistance, please use official support channels.
--->
+> **Note** 
+> Before submitting your issue, please ensure the following:
+> - You are using the latest version of the CLI
+> - You have searched open/closed issues for related posts
+> - You have used the `doctor` command to diagnose common issues
+> - You understand that Github Issues do not adhere to any agreement or SLA
+>   - If you require immediate assistance, please use official support channels
 
 ### Summary
+<!-- Short summary of what is going on or to provide context -->
 
-_Short summary of what is going on or to provide context_.
-
-### Steps To Reproduce:
+### Steps To Reproduce
 
-**Repository to reproduce:** [dreamhouse-lwc](https://github.com/dreamhouseapp/dreamhouse-lwc)
+> **IMPORTANT**
+> Provide a repository that is configured to reproduce the issue. If you are unable to provide a repo, please explain why not. The more info we have from the start, the faster your issue can be resolved.
+> Your issue may be closed if proper instructions are not included.
+>
+> - Generate a project with `sf project generate` or fork [dreamhouse-lwc](https://github.com/trailheadapps/dreamhouse-lwc)
+> - Provide detailed step-by-step instructions on how to reproduce the issue
 
-***NOTE:** If your issue is not reproducable by dreamhouse-lwc, i.e. requires specific metadata or files, we **require** a link to a simple Salesforce project repository with a script to setup a scratch org that reproduces your problem.*
-
-1.  This is step 1.
-1.  This is step 2. All steps should start with '1.'
 
 ### Expected result
-
-_Describe what should have happened_.
+<!-- Describe what should have happened -->
 
 ### Actual result
-
-_Describe what actually happened_.
+<!-- Describe what actually happened -->
 
 ### System Information
+<!-- Which shell/terminal are you using? (bash, zsh, powershell 7, cmd.exe, etc) -->
+<!-- Paste the **full** output of the `version --verbose --json` command below -->
```

Comment:
suggestion: does doctor provide anything better that the version --verbose doesn't that we might want?